src/components/LocationPage/LocationPageBlock.style.ts

```diff
@@ -5,24 +5,24 @@ import {
 } from 'assets/theme/sizes';
 
 export const BlockContainer = styled.div`
-  margin: 3rem 1rem 0;
+  margin: 0 1rem 4.5rem;
```

Comment:
when unifying the location page spacing, i spaced each section via the top margin. this moves the spacing to the bottom margin instead (and increases it a little bit in accordance with the location page mocks)